src/com/cdd/bao/template/ModelSchema.java

```diff
@@ -274,6 +276,7 @@ private void setupResources()
 		isExclude = model.createProperty(PFX_BAT + IS_EXCLUDE);
 		isWholeBranch = model.createProperty(PFX_BAT + IS_WHOLEBRANCH);
 		isExcludeBranch = model.createProperty(PFX_BAT + IS_EXCLUDEBRANCH);
+		isContainer = model.createProperty(PFX_BAT + IS_CONTAINER);
```

Comment:
This could also be managed using an `EnumMap`. That would allow rewriting the quadruple ternary statements more like map lookups.

Reply:
I'm curious to first here Alex's remarks here.  I actually like these line explicit here because it clues maintainers into what exactly we are looking for in the ontologies.  IE, we sacrifice on compactness, but we gain something in self-commenting code.